### ISSUE TITLE
SLES 12 error installing rsync-bpc-3.1.2.0

### ISSUE THREAD

**Issue by mschlawin** [backuppc/backuppc#258]
I apologize if this is the wrong place to post this, but the other links I tried appear to be down.
I have been running BackupPC 3.3.1 on SLES 11 for years.  I just upgraded the server to SLES 12 SP4 and everything appeared to work.  I then tried to upgrade BackupPC to 4.3 and have an install problem similar to some other posts I have read.

First I did this:
cd backuppc-xs/
perl Makefile.PL
make
make test
make install

and that seemed to work.

Then I tried to install rsync-bpc.  The configure command showed successful, but had these lines:
checking for _acl... no
checking for __acl...no
checking for _facl...no
checking for __facl...no
checking whether to support ACLS..running tests:
checking for acl_get_fine in -lacl...no
checking for ACL support...no
checking ACL test results...No ACL support found

So it is no surprise that when I run make, I get
Makefile:65:  recipe for target 'lib/sysacls.o' failed
make: ***[lib/sysacls.o] Error 1

zypper pa shows that libacl1 2.2.52-6.1 is installed.   I do not see a libacl1-dev package, which I'm guessing is the problem.  

If that is that problem, does anyone know how to get that package for SLES 12 sp4 ?

Thanks

Matt

**Comment by mschlawin** [comment]
OK, I think I got it.  I'll post here just in case there are other SuSE users out there.

Go to download.suse.com and download the correct Software Development Kit for your version of SLES.  Mount that .iso on the server and go to the /suse/x86_64 directory.   There is where I found the libacl-devel .rpm.  When I tried to run it, it failed with a dependency error.  I found the libattr-devel .rpm in the same directory, installed that, and then the libacl-devel rpm installed without a problem.

I re-ran the configure command and this time it said it found the ACL provider, and the make command ran without error.

I'm testing BackupPC now, but it does seem to be working.

Matt
